frontend/actions.js

```diff
@@ -152,8 +152,9 @@ module.exports = ({setState, getState}) => {
   const sendAllFunds = async (state) => {
     setState({calculatingFee: true})
 
+    const address = state.sendAddress.fieldValue
     const balance = await wallet.getBalance()
-    const txFee = await wallet.getAllFundsTxFee()
+    const txFee = await wallet.getAllFundsTxFee(address)
```

Comment:
I would rather put the "state.sendAddress.fieldValue" directly as an input of getAllFundsTxFee, since you would need to look at the definition of "address" anyway if you want to understand the call below and it is used only once